debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlConnectorIT.java

```diff
@@ -121,6 +123,24 @@ public void shouldNotStartWithInvalidConfiguration() {
         assertConnectorNotRunning();
     }
 
+    @Test
+    public void shouldNotStartWithUnknownJdbcDriver() {
+        config = DATABASE.defaultConfig()
+                .with(MySqlConnectorConfig.JDBC_DRIVER, "foo.bar")
+                .build();
+
+        final AtomicBoolean successResult = new AtomicBoolean();
+        final AtomicReference<String> message = new AtomicReference<>();
+        start(MySqlConnector.class, config, (success, msg, error) -> {
+            successResult.set(success);
```

Comment:
Tested using Atomic references here otherwise the test still passes regardless of the assertions within the callback failing. There is another test in this class `shouldNotStartWithInvalidConfiguration` that needs the same treatment. It asserts `assertThat(error).isNotNull();` however this assertion actually fails but the test still passes.